templates/app-ts-esm/test/helper.ts

```diff
@@ -15,7 +15,9 @@ const AppPath = path.join(__dirname, '..', 'src', 'app.ts')
 // Fill in this config with all the configurations
 // needed for testing the application
 async function config () {
```

Comment:
Btw, why `config` function return promises in ts templates?
Should I replace by a sync function?

Reply:
I think because that's the signature.